lib/Graph/Nodes.cpp

```diff
@@ -818,7 +818,8 @@ bool RowwiseQuantizedFullyConnectedNode::verify() const {
 
 bool GatherNode::verify() const {
   bool isValid = checkType(getResult(), getData().getElementType(), this);
-  isValid &= checkType(getIndices(), ElemKind::Int64ITy, this);
+  isValid &= (checkType(getIndices(), ElemKind::Int64ITy, this) ||
```

Comment:
@qcolombet I would like to check that if the type is either of the two I supplier. But it will print out some warnings if one of the check fails, which is undesirable. Do you have any suggestion to allow check for an array of kinds?

Reply:
Resolved.

Reply:
Good :).